pyonfx/ass_core.py

```diff
@@ -413,7 +437,10 @@ def __init__(
         self.__plines = 0
         self.__ptime = time.time()
 
-        self.meta, self.styles, self.lines = Meta(), {}, []
+        self.meta: Meta = Meta()
```

Comment:
Linting should not be needed here, right?

Reply:
Adding typehint to these variables showed the incorrect type in line 1149 though https://github.com/Varde-s-Forks/PyonFX/commit/96d00defadc8c55adc5f8c1423db80bb595b8586